Sources/Utility/ArgumentParser.swift

```diff
@@ -200,6 +203,26 @@ public struct PathArgument: ArgumentKind {
     public static var completion: ShellCompletion = .filename
 }
 
+public struct ExecutableName: ArgumentKind {
```

Comment:
I think these do not belong here because they're not generic enough for the base argument parser library.